src/plugins/eqht-css/_base.scss

```diff
@@ -0,0 +1,20 @@
+.wb-eqht-grd{
+	align-items: stretch;
+	display: flex;
+	flex-wrap: wrap;
+	height: 100%;
+
+	>div[class*="col-"]{
```

Comment:
no need to specify the element "div"

Reply:
Removed div element... No longer specified.